scripts/check_site_lib.lua

```diff
@@ -7,6 +7,28 @@ local function loadvar(varname)
    end
 end
 
+local function array_to_string(array)
+   local string = ''
+   for _, v in ipairs(array) do
+      if #string >= 1 then
+         string = string .. ', '
+      end
+      string = string .. v
+   end
+   return '[' .. string .. ']'
+end
+
+local function assert_one_of(var, array, msg)
+   local pass = false
+   for _, v in ipairs(array) do
+      if v == var then
+         pass = true
```

Comment:
You can just `return` here, then you don't need the `pass` variable. `assert(pass, msg)` could be replaced by `error(msg)` then.